backend/controllers/profile_controller.go

```
@@ -92,130 +96,121 @@ func GetProfile(ctx *gin.Context) {
+		}{rank, lbName, lbUser.EloRating, lbAvatar, lbUser.Email == email})
 		rank++
 	}
 
-	// Fetch debate history
-	debateCursor, err := db.MongoDatabase.Collection("debates").Find(
+	// Fetch user's debate history
+	debatesCursor, err := db.MongoDatabase.Collection("debates").Find(
 		dbCtx,
 		bson.M{"email": email},
-		options.Find().SetSort(bson.M{"date": -1}).SetLimit(5),
+		options.Find().SetSort(bson.D{{"date", 1}}),
```

golangci-lint (1.64.8)

composites: go.mongodb.org/mongo-driver/bson/primitive.E struct literal uses unkeyed fields

(govet)